### PR TITLE
Renames `wandb local` to `wandb server`

### PR DESCRIPTION
**Pull request by jsbroks**
Fixes [WB-9867](https://wandb.atlassian.net/browse/WB-9867)

Description
-----------
Rename wb local to wb server

### CODE REVIEW

File: wandb/cli/cli.py [hunk 1]
```diff
@@ -1214,7 +1214,7 @@ def agent(ctx, project, entity, count, sweep_id):
     #                'args': ['--max_epochs=10']})
 
 
-@cli.command(context_settings=CONTEXT, help="Run the W&B local sweep controller")
+@cli.command(context_settings=CONTEXT, help="Run the W&B server sweep controller")
```
Comment (kptkin) [inline wandb/cli/cli.py:1217]: is this relevant? It is just the sweeps controller...

File: wandb/sdk/wandb_run.py [hunk 2]
```diff
@@ -3180,7 +3180,7 @@ def _footer_local_warn(
             if out_of_date:
                 # printer = printer or get_printer(settings._jupyter)
                 printer.display(
-                    f"Upgrade to the {latest_version} version of W&B Local to get the latest features. "
+                    f"Upgrade to the {latest_version} version of W&B Server to get the latest features. "
                     f"Learn more: {printer.link(wburls.get('upgrade_local'))}",
```
Comment (kptkin) [inline wandb/sdk/wandb_run.py:3184]: should we fix the link name as well? (nit)

**Comment by kptkin** [comment]
found another place where we would probably want to change the name:
https://github.com/wandb/client#docker

File: wandb/cli/cli.py (hunk 1 above)
Comment (raubitsj) [reply wandb/cli/cli.py:1217]: good catch, this is not relevant, it is a different thing.

**Review by raubitsj — CHANGES_REQUESTED** [review]

**Review by dmitryduev — APPROVED** [review]
LGTM!

**Review by raubitsj — APPROVED** [review]

File: wandb/cli/cli.py (hunk 1 above)
Comment (raubitsj) [reply wandb/cli/cli.py]: er.. this is still called local sweep controller.

**Pull request merged**